suppressions.txt

```diff
@@ -0,0 +1,2 @@
+missingInclude
```

Comment:
Why it is required?

Reply:
Some include headers are system (or 3rd party) - suppressed by missingIncludeSystem
"version.hpp" - generated by cmake. Suppressed explicitly by an inline suppression.

Reply:
Is there options to suppress this warning for system includes only?

Reply:
Looks like it works for all arrow brackets includes ( <iosteam>, <crypto>, etc)

Reply:
there're 2 warning types:
missingIncludeSystem and missingInclude

missingIncludeSystem triggers on arrow brackets includes, so it suppresses even aos core lib missing includes.